CONTRIBUTING.md

```diff
@@ -40,5 +40,86 @@ Python code is expected to follow
 [PEP8](https://www.python.org/dev/peps/pep-0008/) and
 [not commit atrocities](https://www.youtube.com/watch?v=wf-BqAjZb8M).
 JavaScript, CSS/Less, and markup should follow our
-[front-end standards](https://github.com/cfpb/front-end).
+[front-end standards](https://github.com/cfpb/development).
 When in doubt, mimic the styles and patterns in the existing codebase.
+
+### Browser support
+
+We configure [Autoprefixer](#autoprefixer) and [Babel](#babel) to support the following list of browsers.
+
+- Latest 2 releases of all browsers including:
+    - Chrome
+    - Firefox
+    - Safari
+    - Internet Explorer
+    - Edge
+    - Opera
+    - iOS Safari
+    - Opera Mini
+    - Android Browser
+    - BlackBerry Browser
```

Comment:
We should stop proclaiming to support Blackberry. Absolutely no one is testing on it, now that we have migrated to the IPhone.